docs/user/installation_and_configuration.rst

```diff
@@ -274,6 +274,27 @@ the same Accumulo instance. You should remove any GeoMesa JARs under
 
 .. _install_geoserver_plugins:
 
+Bootstrapping GeoMesa and Accumulo on Elastic Map Reduce
+--------------------------------------------------------
+
+A script to bootstrap GeoMesa and Accumulo is provided in ``geomesa-tools/emr4`` and on this public S3 bucket: ``s3://elasticmapreduce-geomesa/bootstrap-geomesa.sh``. These rely on the EMR managed Hadoop and ZooKeeper applications. See ``geomesa-tools/emr4/README.md`` for more details. The commands below launch a GeoMesa EMR cluster:
+
+.. code-block:: bash
+    NUM_WORKERS=2
```

Comment:
I believe you need an empty line between the code-block directive and the first line of the block.